samples/python_interop/generating_n_random_bits/README.md

```diff
@@ -0,0 +1,5 @@
+# A simple Q# program to generate an array of N random bits and Python program to count the number of 1s returned in the result array
```

Comment:
The core of this sample is what it illustrates (Python interop) rather than what exactly it does, so I'd focus on that:
```suggestion
# Passing arguments from Python to Q# and processing measurement results in Python
```